scripts/e2e_setup_cluster.sh

```diff
@@ -126,6 +131,8 @@ echo "## generating K8 api flags files"
 generate_k8_api_data
 echo "## start Kind cluster with precreated CA key/cert"
 create_cluster
```

Comment:
Can we delete cluster before trying to create it ? Might be good to remove any stale clusters on self hosted
```
kind delete cluster
```
It is idempotent.